.github/workflows/publish-npm-packages.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish NPM Packages
+on:
+  workflow_dispatch:
+    inputs:
+      ampversion:
+        description: 'AMP version'
+        required: true
+      tag:
+        description: 'NPM package tag (latest | nightly)'
+        required: true
+jobs:
+  publish:
+    environment: NPM_TOKEN #TODO: change environment name in repo settings
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        extension:
+          [
+            'amp-selector',
+            'amp-accordion',
+            'amp-date-countdown',
+            'amp-date-display',
+            'amp-fit-text',
+            'amp-inline-gallery',
+            'amp-instagram',
+            'amp-lightbox',
+            'amp-timeago',
+            'amp-youtube',
+          ]
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.events.inputs.ampversion }}
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+          registry-url: https://registry.npmjs.org
```

Comment:
Instead of installing a hard-coded version of `node` here, it may be more robust to use the latest version that's already provided by GH actions.

For our other configs, we run this wrapper script that sets up the node environment and installs packages. Maybe it can be reused here (with a change that doesn't install packages)?

```yaml
      - name: Install Dependencies
        run: bash ./.github/workflows/install_dependencies.sh
```

Reply:
I'll use `check-latest` to always use the latest.